src/github_runner_manager/manager/runner_scaler.py

```diff
@@ -126,7 +126,11 @@ def reconcile(self, quantity: int) -> int:
 
         if self._reactive_config is not None:
             logger.info("Reactive configuration detected, going into experimental reactive mode.")
-            return self._reconcile_reactive(quantity)
+            return reactive_runner_manager.reconcile(
+                quantity=quantity,
```

Comment:
I was trying to understand what this quantity means and it wasn't immediately obvious. Perhaps something like `expected_quantity` or something that indicates that this number is how many runners are required

Reply:
I realise this has been the name for a while, just came to mind now

Reply:
Updated.